src/commands/capes.js

```diff
@@ -0,0 +1,105 @@
+const embedUtils = require('../utils/embed.js')
+const uuidUtils = require('../utils/uuid.js')
+const axios = require('axios').default
+
+/**
+ *
+ * @param {Client} client
+ * @param {Message} message
+ * @param {string[]} args
+ */
+module.exports = {
+    name: "capes",
+    description: "Edit capes",
+    options: [
+        {
+            name: "pull",
+            description: "Pull capes",
+            type: 1,
+            options: [
+                {
+                    name: "force",
+                    description: "Force overwrite",
+                    type: 5,
+                }
+            ]
+        },
+        {
+            name: "push",
+            description: "Push capes",
+            type: 1
+        },
+        {
+            name: "add",
+            description: "Add capes",
+            type: 1,
+            options: [
+                {
+                    name: "user_id",
+                    description: "User ID",
+                    type: 4,
+                    required: true
+                },
+                {
+                    name: "minecraft_username",
+                    description: "Minecraft username",
+                    type: 3,
+                    required: true
+                },
+                /*{
+                    name: "cape",
+                    description: "Cape",
+                    type: 3,
+                    required: true,
+                    choices: [
+                        {
+                            name: "Contributor",
+                            name: "Contributor cape",
+                            value: ""
+                        },
+                    ]
+                }*/ //anal sex
+            ]
+        }
+    ],
+
+    run: async (interaction) => {
+
+        for (let index of interaction.options.data) {
+            switch (index.name) {
+                case "pull": {
+                    const isForced = interaction.options.getBoolean("force") ?? false
+                    const pullResult = await capeUtils.pull(isForced);
+                    if (!pullResult && !isForced) return interaction.reply({ embeds: [embedUtils.error('Add `force:true` in the options to override local data.')] });
+                    return interaction.reply({ embeds: [embedUtils.success("Pulled!")] });
+                }
+                case "push": {
+                    const pushResult = await capeUtils.push();
+                    if (!pushResult) { await capeUtils.pull(); await capeUtils.push(); }
+                    return interaction.reply({ embeds: [embedUtils.success('Pushed to remote.')] })
+                }
+                case "add": {
+                    const minecraft_username = await interaction.options.getString("minecraft_username")
+                    const user = await interaction.options.getInteger("user_id")
+                    try {
+                        const _ = await axios.get(`https://discordapp.com/api/v9/users/${user}`, {
```

Comment:
avoid doing request here, move this into a utils file

Reply:
It's useless to create a file for 4 lines of code

Reply:
Fixed